.gitlab/build-deb-rpm.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+if [ -n "$CI_COMMIT_TAG" ] && [ -z "$RUBY_PACKAGE_VERSION" ]; then
+  RUBY_PACKAGE_VERSION=${CI_COMMIT_TAG##v}
+fi
+
+source common_build_functions.sh
+
+# Get a list of directories with the "-static" suffix
+directories=($(find ../vendor -maxdepth 4 -type d -name "*-static"))
+
+# Check if any directories were found
+if [ ${#directories[@]} -eq 0 ]; then
+  echo "No directories with the '-static' suffix found."
+  exit 1
+fi
+
+# Loop through the directories
+for dir in "${directories[@]}"; do
+  # Remove the "./" prefix and the "-static" suffix from the directory name
+  new_dir="${dir#./}"
+  new_dir="${new_dir%-static}"
+  # Rename the directory
+  mv "$dir" "$new_dir"
```

Comment:
Minor: I wonder if instead of moving, we could symlink them instead. That seems like it may make the gems work with both `-static` and non-static Rubies...?